app/controllers/users_controller.rb

```diff
@@ -4,6 +4,7 @@ def show
 
     github_results = GithubResults.new
     @repos = github_results.get_repos(current_user.token)[0..4]
+    @followers = github_results.get_followers(current_user.token)[0..4]
```

Comment:
It makes sense to only bring in 5 followers, but the user story doesn't say anything about limiting the number of followers shown that I can see.